invertedai/common.py

```diff
@@ -124,9 +128,13 @@ class AgentAttributes:
 
     length: float  #: Longitudinal extent of the agent, in meters.
     width: float  #: Lateral extent of the agent, in meters.
-    rear_axis_offset: float  #: Distance from the agent's center to its rear axis in meters. Determines motion constraints.
+    #: Distance from the agent's center to its rear axis in meters. Determines motion constraints.
```

Comment:
Did you check that this renders correctly in sphinx?

Reply:
Yes, it does.